.vib/drupal/cypress/cypress/integration/drupal_spec.js

```diff
@@ -0,0 +1,15 @@
+/// <reference types="cypress" />
+import { random } from '../support/utils';
+
+it('allows creating an article with an image', () => {
+  cy.login();
+  cy.visit('/node/add/article');
+  cy.fixture('articles').then((article) => {
+    cy.get('.text-full[name*=title]').type(`${article.newArticle.title} ${random}`);
+    cy.get('[type=file]').get('[type=file]').selectFile('cypress/fixtures/images/test_image.jpeg', { force: true });
```

Comment:
This double `get` is really weird. I'm not sure if the intention was to have something similar to [cy.within](https://docs.cypress.io/api/commands/within) but here a single `get` should suffice. Checking how Cypress behaves, it seems it checks 2 times for that same selector, which is redundant. Please try using the following:

```suggestion
    cy.get('[type=file]').selectFile('cypress/fixtures/images/test_image.jpeg', { force: true });
```

Reply:
I didn't notice. Anyway. It's fixed now: 

https://github.com/mdhont/charts/actions/runs/3051810313